tardis/montecarlo/base.py

```diff
@@ -500,18 +495,20 @@ def montecarlo_virtual_luminosity(self):
             self._montecarlo_virtual_luminosity[:-1]
             / self.time_of_simulation.value
         )
+        
+    @property
+    def nthreads(self):
+        return(tardis.io.config_reader.Configuration.montecarlo.nthreads)
```

Comment:
pretty sure that this is wrong

Reply:
Hi yes sorry, this was just a draft PR at that time so my changes weren't final. Please review now.